docs/spec/mint/state.md

```diff
@@ -26,6 +27,7 @@ type Params struct {
 	InflationMax        sdk.Dec // maximum inflation rate
 	InflationMin        sdk.Dec // minimum inflation rate
 	GoalBonded          sdk.Dec // goal of percent bonded atoms
+	BlocksPerYear       int64   // expected blocks per year
```

Comment:
`uint64` perhaps? We'll never have negative blocks per year.

Reply:
++

Reply:
okay using uint64 now